traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -114,11 +117,20 @@ func ReadDSSHandler(w http.ResponseWriter, r *http.Request) {
 
 	dss := TODeliveryServiceServer{}
 	dss.SetInfo(inf)
-	results, err := dss.readDSS(inf.Tx, inf.User, inf.Params, inf.IntParams, nil, nil)
+	results, err, maxTime := dss.readDSS(nil, inf.Tx, inf.User, inf.Params, inf.IntParams, nil, nil, false)
 	if err != nil {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, err)
 		return
 	}
+	if maxTime != nil {
```

Comment:
There are conflicts in this file now due to the total reworking of the servers API here: https://github.com/apache/trafficcontrol/pull/4700/files